### ISSUE TITLE
Recent Releases for Science Transformers

### ISSUE THREAD

**Issue by gordonwatts** [ssl-hep/ServiceX#666]
# Story
_As a physicists I want to use the most recent releases of AnalysisBase so I can take advantage of modern ATLAS `xAOD` code._

# Assumptions
1. Just a matter of researching the most recent version of everything, before the switch to AL9 containers.

# Acceptance Criteria
1. R21, R22, and R24 will be available on ServiceX.

# To do:

* [x] Find the most recent versions of all images, and create them in the `science containers` repo
* [x] Make sure R21, R22, R24 code generator backends are available in SX.

**Comment by gordonwatts** [comment]
Looking at the ATLAS [container registry](https://gitlab.cern.ch/atlas/athena/container_registry/8439?orderBy=NAME&sort=desc&search[]=21.2.2&search[]=) - which does not have a great interface.

| Release Series | Last Container Tag |
| --- | --- |
| 21 | 21.2.266 |
| 22 | 22.2.113 |
| 24 | 24.2.28-centos7, 24.2.26 |

Ops - it looks like the change to how ATLASis tagging things as it moves forward with AL9 might cause our scripts some "issues:"! So - let's take the last non-AL9 set of builds, and we can do the AL9 stuff in a separate check to ensure everything works.

**Comment by gordonwatts** [comment]
When I ran locally on my (windows!) laptop, with the following in my `values.yaml` file, the new code generators for R24 (and PHYSLITE) worked just fine;

```yaml
codeGen:
  atlasr24:
    enabled: true
    image: sslhep/servicex_code_gen_func_adl_xaod
    pullPolicy: Never
    tag: develop
    defaultScienceContainerImage: sslhep/servicex_func_adl_xaod_transformer
    defaultScienceContainerTag: 24.2.26
  atlasr22:
    enabled: true
    image: sslhep/servicex_code_gen_func_adl_xaod
    pullPolicy: Never
    tag: develop
    defaultScienceContainerImage: sslhep/servicex_func_adl_xaod_transformer
    defaultScienceContainerTag: 22.2.113
  atlasr21:
    enabled: true
    image: sslhep/servicex_code_gen_func_adl_xaod
    pullPolicy: Never
    tag: develop
    defaultScienceContainerImage: sslhep/servicex_func_adl_xaod_transformer
    defaultScienceContainerTag: 21.2.231
```

**Comment by gordonwatts** [comment]
This story can be closed - once other PR's like #672 get merged.

**Comment by ponyisi** [comment]
We can close, then?
